django_pydantic_field/base.py

```diff
@@ -83,7 +87,14 @@ def wrap_schema(
     return pydantic.create_model(type_name, **params)
 
 
-def extract_export_kwargs(ctx: dict, extractor=dict.get):
+def prepare_schema(schema: "ModelType", owner: t.Any = None) -> None:
+    if not getattr(schema, _RESOLVED_FORWARD_REF_FLAG, False):
+        namespace = get_local_namespace(owner)
+        schema.update_forward_refs(**namespace)
+        setattr(schema, _RESOLVED_FORWARD_REF_FLAG, True)
```

Comment:
This may fail if model (more likely a `pydantic.dataclass`) has `slots=True`. Need to investigate that option